apps/hash-graph/lib/graph/src/store/query/filter.rs

```diff
@@ -275,7 +275,6 @@ pub enum Parameter<'p> {
     Boolean(bool),
     Number(f64),
     Text(Cow<'p, str>),
-    #[serde(skip)]
     Any(Value),
```

Comment:
Just a quick question about the ordering when deserializing. Would serde know to not make values of the `Any` variant when it acts like a match-all? Or does it try from top down? The order here makes sense in relation to top-down, I'm just curious :)

Reply:
It’s top-down. For the deserialization we only need `Any` as the parameter will be converted anyway, but for predefined queries (defined on `Filter`) it`s easier to have all types available, so we can skip conversion.

Reply:
Oh interesting. Thanks for explaining